### PR TITLE
Add Cask postico1

### PR DESCRIPTION
**Pull request by razvanazamfirei**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask --online <cask>` is error-free.
- [X] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [X] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-versions/search?q=is%3Aclosed&type=Issues).
- [X] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [X] `brew audit --new-cask <cask>` worked successfully.
- [X] `brew install --cask <cask>` worked successfully.
- [X] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by reitermarkus** [comment]
Are you using this yourself, or simply adding this because of https://github.com/Homebrew/homebrew-cask/pull/144285?

**Comment by razvanazamfirei** [comment]
@reitermarkus Just in response to https://github.com/Homebrew/homebrew-cask/pull/144285

**Comment by reitermarkus** [comment]
Generally we don't proactively add anything to this repo. If someone actually needs it they can submit it.

File: Casks/postico1.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  livecheck do
+    url "https://eggerapps.at/postico/download/"
+    strategy :extract_plist
+  end
+
+  conflicts_with cask: "postico"
+
+  # Rename to prevent confusion with main postico cask
+  app "Postico.app", target: "Postico #{version.major}.app"
```
Comment (reitermarkus) [inline Casks/postico1.rb:18]: This shouldn't be needed since there is also a `conflicts_with`.

**Comment by pawelad** [comment]
I'd very much appreciate this : )

I bought a license for Postico 1 and will probably not upgrade to Postico 2 any time soon. Homebrew just updated my local version from 1 to 2, so as far as I understand, I need to uninstall it altogether and install Postico 1 outside of Homebrew?

**Pull request closed without merging**